docs/WindowsSubsystemForLinux.md

```diff
@@ -0,0 +1,94 @@
+
+No special configuration is needed to run CE under WSL. Some configuration is required for hosting the Microsoft Visual C++ ("MSVC") compiler. Testing has mainly been done on the Ubuntu distro but any distro should work.
+
+## WSL/Windows interop and its limitations
+WSL offers rich interop with Windows processes. You can run any Windows executable, such as "`cl.exe`", from a bash shell. But this interop capability has some limitations. 
+- Windows volumes: While Windows executables can be run from bash, they cannot see Linux volumes. Windows executables that need to read or write files must be run on a Windows volume. This means all MSVC compiles must be done in the Windows `%TEMP%` directory instead of in the `bash` environment's temp directory. 
+- Path: The WSL path set in bash prepends the Windows path. While Linux filesystems support odd naming conventions such as spaces and parentheses, Windows' path uses these as a matter of course (e.g., `c:\Program Files (x86)`). Additionally, the Windows path delimiter is `\` instead of `/`, and it uses drive letters instead of mount points that are separated with a colon. 
+- Path names: A Windows path of `c:\tmp` is normally referred to as `/mnt/c/tmp` in `bash`. However, users can customize their `drvfs` mount points. A tool is provided in newer Windows releases, `/bin/wslpath`, that will convert paths between systems. Code in CE currently does the conversion between the standard conventions using string manipulation. 
+- Environment variables: While the Windows path is available in bash, Windows environment variables are not. CE uses `cmd.exe /c echo %TEMP%` to determine the Windows temporary directory.
+- Execution environment: The execution environment cannot currently be set when doing `childprocess.spawn`. This is a serious issue for the MSVC compiler, which is highly environment-dependent (e.g., `%INCLUDE%`, `%LIBPATH%`, etc.)
+
+## Configuration
+This section is intended for the many WSL users who are new to Linux. 
+
+If you plan on debugging CE, you should clone the CE repo on a Windows volume.
+
+CE is built on node.js ("node"). The easiest way to install node is using NVM, the Node Version Manager. Run the following commands from a bash shell:
+- `apt-get update` to make sure apt is up-to-date
+- `apt-get install build-essential libssl-dev`, though you probably have these already
+- Check https://github.com/creationix/nvm/releases for the latest NVM release, substituting it in the next command.
+- `curl https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh | bash` to install NVM
+- `source ~/.profile` to reload your profile, bringing NVM into your environment
+- `nvm ls-remote -lts` to show the latest supported version of node.js
+- `nvm install 8.9.3`, substituting the latest version, to install node.js
```

Comment:
It's probably better if you don't suggest a "latest version". We only guarantee that it works on the latest LTS version (Even tho there are probably no reasons for it not to work further down the line)

Reply:
I'm suggesting you find the LTS on the line above--should I remove all mention of latest, or reword to "latest LTS"?